_posts/2022-02-28-python-shorts-unit-testing.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Python Shorts: The Benefits of Unit Testing"
+author:
+  display_name: Jake Adams
+  email: jdadams@utah.gov
+date: 2021-02-28 11:00:00
```

Comment:
**chore**: we'll need to freshen up this date and the file name date when you are ready for a merge